packages/flutter/test/widgets/shortcuts_test.dart

```diff
@@ -1255,6 +1257,50 @@ void main() {
       expect(invoked, 1);
       invoked = 0;
     });
+
+     testWidgets('Shortcuts does not insert a semantics node when includeSemantics is false', (WidgetTester tester) async {
```

Comment:
I think the indentation might be over by 1 space here?

Reply:
Removed! Thank you for catching that.